api/nakadi-event-bus-api.yaml

```diff
@@ -2328,6 +2386,62 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+  EventTypeAuthorization:
+    type: object
+    description: |
+      Authorization section for an event type. This section defines three access control lists: one for producing
+      events ('write'), one for consuming events ('read'), and one for administering an event type ('admin').
+      Regardless of the values of the authorization properties, administrator accounts will always be authorized.
+    properties:
+      admin:
+        type: array
```

Comment:
If we go down that route, I would prefer something like `admin_attributes`, `read_attributes`, and `write_attributes`, which is shorter and more precise (these are lists of authorization attributes). But I still think, imho, that `admin`, `read`, and `write` are sufficient, and clearer.